modules/networkelb-dashboard/main.tf

```diff
@@ -0,0 +1,265 @@
+terraform {
+  required_providers {
+    lightstep = {
+      source  = "lightstep/lightstep"
+      version = "~> 1.60.2"
+    }
+  }
+  required_version = ">= v1.0.11"
+}
+
+resource "lightstep_metric_dashboard" "aws_networkelb_dashboard" {
+  project_name   = var.lightstep_project
+  dashboard_name = "AWS NETWORKELB"
+
+  chart {
+    name = "Healthy Host Count"
+    rank = "0"
+    type = "timeseries"
+
+    query {
+      query_name = "d"
```

Comment:
Interesting ... why does the first query begin with "d" in this case, but the first query of every other chart is "a"?